src/load-script.test.ts

```diff
@@ -47,7 +47,7 @@ describe("loadScript()", () => {
             '<script src="https://www.paypal.com/sdk/js?client-id=test"></script>';
         window.paypal = paypalNamespace;
 
-        const response = await loadScript({ "client-id": "test" });
+        const response = await loadScript({ clientID: "test" });
```

Comment:
I prefer camelCase, especially for object keys, but now this raises the question if it should be `clientID` or `clientId`.

Reply:
Great point @bywood. It seems like we've come to an agreement to use `clientId` format. This follows the naming conventions that Google has detailed in their style guide: https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined